examples/linear_elasticity/test_elasticity.py

```diff
@@ -173,6 +173,33 @@ def right_boundary(x):
         analytical_stress
     )
 
+    law_3d = LinearElasticityModel(
+        parameters={"E": youngs_modulus, "nu": poissons_ratio},
+        constraint=Constraint.FULL,
+    )
+    u_3d = df.fem.Function(V)
+    problem_3d = IncrSmallStrainProblem(
+        law_3d,
+        u_3d,
+        [bc_left, bc_right],
+        1,
+        solver_constraint=Constraint.UNIAXIAL_STRAIN,
+    )
+    solver_3d = NewtonSolver(MPI.COMM_WORLD, problem_3d)
+    n, converged = solver_3d.solve(u_3d)
```

Comment:
n and converged are not used in the following, maybe delete
I'm always for some (more) comments: At least two words what is the test about. And in this case the to options you test